Dockerfile

```diff
@@ -14,12 +14,11 @@ RUN npm set progress=false && npm config set depth 0 && \
     npm install
 
 # Run lint and tests
-RUN npm run lint
-RUN npm run test
```

Comment:
er det ikke meget rart at have dem i to tempi ? altså først test og så lint - jeg tror det bliver nemmere at læse